cypress/components/menu/menu.cy.tsx

```diff
@@ -1818,10 +1393,10 @@ context("Testing Menu component", () => {
       }
     );
 
-    it.each([250, 750, 1350])(
+    it.each(["default", "large"] as MenuDividerProps["size"][])(
       "should pass accessibility tests for Menu when size is %spx",
       (size) => {
-        CypressMountWithProviders(<MenuComponent size={size} />);
+        CypressMountWithProviders(<MenuDividerComponent size={size} />);
```

Comment:
Is there a reason why `MenuComponent` has been swapped with `MenuDividerComponent` here?

Reply:
Yes, I made "MenuDividerComponent " as a Custom component which belongs to size prop only.